source/GameMode.cpp

```diff
@@ -382,11 +382,10 @@ void GameMode::updateHealth() {
 	auto& breaches = ship->getBreaches();
 
 	// Breach health drain
-	for (auto& breache : breaches) {
+	for (auto& breach : breaches) {
```

Comment:
nit: can we make this a `const auto&`? Since it's a smart pointer, it's good form to get the pointer itself as a `const` reference since we don't plan on reassigning the pointer.

Reply:
sure